client/src/js/services/PrototypeApiService.js

```diff
@@ -244,17 +244,32 @@ function PrototypeApiService($http, util) {
 
     // append the id to the base url
     var target = this.url.concat('reports/', param);
+    return reportBuilder(this, param, filetype);
```

Comment:
This `return` will now error.  `reportBuilder()` only takes in two parameters, `reportBuilder(param, filetype)`.  Please update this line to prevent it!